docs/administration/upgrade/_index.md

```diff
@@ -13,8 +13,9 @@ Since the migration might alter the database schema and the settings of `harbor.
 ## Notes
 
 - Again, you MUST backup your data before any data migration.
-- In version 1.10.0, some containers are started by `non-root`. This does not pose problems if you are upgrading an officially released version of Harbor, but if you have deployed a customized instance of Harbor, you might encounter permission issues.
+- In version 1.10.0, some containers are started by `non-root`. This does not pose problems if you are upgrading an ially released version of Harbor, but if you have deployed a customized instance of Harbor, you might encounter permission issues.
```

Comment:
typo?

Reply:
oh yes! weird, i dont know how that happened. thanks for raising!

Reply:
should be fixed now